src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,20 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    void insertDrivers(Car car);

```

Comment:
These methods should be only on the service layer: `void addDriverToCar(Driver driver, Car car);`

Reply:
seeems to be done